### DISCUSSION TITLE
What is the difference to notify?

### DISCUSSION THREAD

**Discussion by Vad1mo** [containrrr/shoutrrr#214] [Q&A] [answered]
When looking for an apprise like framework, I came across you guys and https://github.com/nikoksr/notify
Now I am wondering how are the projects related in terms of functionality, use cases and target audience. 

maybe some comments from the authors of this library would help users like me to ease the decision making.

**Comment by piksel** [comment, marked as answer]
I think the main difference is Shoutrrrs intent to be configured from a third party end user through service-URLs. This is what adds the bulk of the complexity of our project at least.
